src/djcytoscape/templates/djcytoscape/generate_new_form.html

```diff
@@ -1,19 +1,25 @@
 {% extends "djcytoscape/base.html" %}
 {% load crispy_forms_tags %}
 {% block heading_inner %}Generate a New Quest Map{% endblock%}
+
+{% block head %}
+  {{ form.media }}
+{% endblock %}
+
 {% block content %}
+
     {% if interlink %}
         <p>This quest map doesn't exist yet, but you can create it here.  Students who attempt to click the link that
         took you here will see a Page not found (404) error page until the map is created.</p>
 
         <p>Additional settings can be edited after the map has been generated.</p>
```

Comment:
Is this still true?  Create and Edit should pretty much be using the same form now, no?

Reply:
interlinked urls still works

is_primary_scape and autobreak are the additional settings can be edited after the map has been generated. They cant be set in generate_map_form